tests/beta/test_default_tensor.py

```diff
@@ -126,8 +126,15 @@ def prep_par(par, op):
     return par
 
 
-class TestAuxillaryFunctions:
-    """Test auxillary functions."""
+def edges_valid(dev, num_nodes):
+    """Returns True if the edges in a device are properly accounted for, when there are num_nodes in tensor network"""
+    node_edges = [dev._nodes['state'][idx].edges for idx in range(num_nodes)]
+    node_edges_set = set([edge for sublist in node_edges for edge in sublist])
+    return node_edges_set == set(dev._terminal_edges)
```

Comment:
Out of curiosity: is the time taken to run these test affected by using this function? I assume that for these smaller tensor networks it's not too significant

Reply:
Didn't notice any significant slowness of the tests while developing